### PR TITLE
Release/production mainnet deployment

### PR DESCRIPTION
**Pull request by KohliSuraj**
Changes for main net deployment

### CODE REVIEW

**Review by RojhatToptamus — APPROVED** [review]

File: deploy/006_deploy_uma_emp_creator.js [hunk 1]
```diff
@@ -8,8 +8,8 @@ const func = async function (hre) {
     // Deploy our own Finder
     // const Finder = await deployments.get("Finder");
 
-    // Using Finder deployed by UMA on goerli
-    const Finder = { address: "0xE60dBa66B85E10E7Fd18a67a6859E241A243950e" };
+    // Using Finder deployed by UMA on mainnet
+    const Finder = { address: "0x40f941E48A552bF496B154Af6bf55725f18D77c3" };
```
Comment (ShubhamChndrvnshi) [inline deploy/006_deploy_uma_emp_creator.js]: Test case is failing due to changed finder address

Comment (KohliSuraj) [reply deploy/006_deploy_uma_emp_creator.js]: Nice catch :D I just fixed this

Comment (ShubhamChndrvnshi) [reply deploy/006_deploy_uma_emp_creator.js]: I was thinking to do this:
```
const { getAddress } = require("@uma/contracts-node")
    let chainId;
    if (hre.network.config.forking) { 
        const hardhatForkProvider = new hre.ethers.providers.JsonRpcProvider(hre.network.config.forking.url);
        chainId = await hardhatForkProvider.send("eth_chainId")
    }
    
    const Finder = { address: getAddress("Finder", await hre.getChainId()) };
```

Comment (KohliSuraj) [reply deploy/006_deploy_uma_emp_creator.js]: nice solution! I don't think we need this right now, we are only supporting main net for now.

**Review by ShubhamChndrvnshi — APPROVED** [review]

**Pull request merged**